src/chatClient.cpp

```diff
@@ -1852,6 +1855,7 @@ promise::Promise<void> GroupChatRoom::decryptTitle()
         wptr.throwIfDeleted();
         KR_LOG_ERROR("Error decrypting chat title for chat %s:\n%s\nFalling back to member names.", karere::Id(chatid()).toString().c_str(), err.what());
         makeTitleFromMemberNames();
+        return err;
```

Comment:
I'm agree with Sergio. It's caller responsability to manage the promise-fail. GroupChatRoom::decryptTitle() doesn't know how it is going to be used